Content.Shared/Movement/EntitySystems/SharedMobMoverSystem.cs

```diff
@@ -1,23 +1,34 @@
+using Content.Shared.CCVar;
 using Content.Shared.Movement.Components;
+using Robust.Shared.Configuration;
 using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
 using Robust.Shared.Maths;
 using Robust.Shared.Physics;
-using Robust.Shared.Physics.Collision;
 using Robust.Shared.Physics.Dynamics;
 
 namespace Content.Shared.Movement.EntitySystems
 {
     public sealed class SharedMobMoverSystem : EntitySystem
     {
+        private bool _pushingEnabled;
+
         public override void Initialize()
         {
             base.Initialize();
             Get<SharedPhysicsSystem>().KinematicControllerCollision += HandleCollisionMessage;
+            IoCManager.Resolve<IConfigurationManager>().OnValueChanged(CCVars.MobPushing, PushingToggle, true);
+        }
+
+        private void PushingToggle(bool value)
```

Comment:
I mean, this should probably be named "SetPushing" or "SetPushingAllowed" or something.